mypy/main.py

```diff
@@ -1291,6 +1297,10 @@ def set_strict_flags() -> None:
     special_opts = argparse.Namespace()
     parser.parse_args(args, SplitNamespace(options, special_opts, "special-opts:"))
 
+    # Parse extra plugins passed via cli args
+    if special_opts.cli_plugins:
+        options.plugins.extend(special_opts.cli_plugins)
```

Comment:
Users might expect a CLI option with the same name as a config file option to override the config file value.  If this option is going to extend the list of plugins, a name like `--extend-plugins` might be clearer.